workspaces/arborist/lib/diff.js

```diff
@@ -124,6 +135,16 @@ const getAction = ({ actual, ideal }) => {
     return 'CHANGE'
   }
 
+  if (ideal.isLink) {
+    if (packLinks && isSymlink(ideal.path)) {
```

Comment:
this should likely test that `ideal.path` is not a directory rather than it _is_ a symlink